context/app/static/js/components/Detail/SummaryItem.jsx

```diff
@@ -11,15 +11,20 @@ const VerticalDivider = styled(Divider)`
   align-self: center;
 `;
 
+const StyledDiv = styled.div`
+  display: flex;
+`;
+
 function SummaryItem(props) {
-  const { children } = props;
+  const { children, statusIcon } = props;
   return (
-    <>
+    <StyledDiv>
+      {statusIcon && statusIcon}
```

Comment:
Simplify? Perhaps just
```suggestion
      {statusIcon}
```
or maybe
```suggestion
      {Boolean(statusIcon) && statusIcon}
```
if `statusIcon` could be 0?